src/main/java/me/botsko/prism/wands/Wand.java

```diff
@@ -8,66 +8,17 @@
 
 public interface Wand {
 
-	/**
-	 * 
-	 */
-	void playerLeftClick(Player player, Location loc);
-
-	/**
-	 * 
-	 */
-	void playerRightClick(Player player, Location loc);
-
-	/**
-	 * 
-	 * @param player
-	 * @param entity
-	 */
-	void playerRightClick(Player player, Entity entity);
-
-	/**
-	 * 
-	 * @param given
-	 */
-	void setItemWasGiven(boolean given);
-
-	/**
-	 * 
-	 * @return
-	 */
-	boolean itemWasGiven();
-
-	/**
-	 * 
-	 * @param mode
-	 */
-	void setWandMode(String mode);
-
-	/**
-	 *
-	 */
-	String getWandMode();
-
-	Material getItem();
-
-	void setItem(Material material);
-
-	/**
-	 * 
-	 * @param key
-	 */
-	void setItemFromKey(String key);
-
-	/**
-	 * 
-	 * @param item
-	 */
-	void setOriginallyHeldItem(ItemStack item);
-
-	/**
-	 * 
-	 * @param player
-	 */
-	void disable(Player player);
+    void playerLeftClick(Player player, Location loc);
+    void playerRightClick(Player player, Location loc);
+    void playerRightClick(Player player, Entity entity);
+    void setItemWasGiven(boolean given);
+    boolean itemWasGiven();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['METHOD_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)